apps/createsamples/utility.cpp

```diff
@@ -1270,7 +1270,7 @@ void cvCreateTrainingSamples( const char* filename,
                 if( showsamples )
                 {
                     cvShowImage( "Sample", &sample );
-                    if( cvWaitKey( 0 ) == 27 )
+                    if( (cvWaitKey( 0 ) & 0xFF) == 27 )
```

Comment:
Some more investigation shows that ESC is not a printable char, and thus there is no sequence assigned to that. We will have to stick to 27 or define an enum for non printable ASCI values, which could be interesting to do so.